README.md

```diff
@@ -61,42 +61,19 @@ Then run any of the `bash` scripts at the root of this project to send events to
 
 For some of our external API calls we use `openapi-generator` to generate the models used in the API clients. The Open API specifications used can be found in directory `openapi-specs`.
 
-## Updating the Open API specs
-
-Run the following commands to take a copy of the latest specs:
-
-```
-curl https://nomis-prisoner-api-dev.prison.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/nomis-prisoner-api-docs.json
-curl https://nomis-sync-prisoner-mapping-dev.hmpps.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/nomis-mapping-service-api-docs.json
-curl https://activities-api-dev.prison.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/activities-api-docs.json
-curl https://manage-adjudications-api-dev.hmpps.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/adjudications-api-docs.json
-curl https://adjustments-api-dev.hmpps.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/sentencing-adjustments-api-docs.json
-curl https://incident-reporting-api-dev.hmpps.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/incidents-api-docs.json
-curl https://csip-api-dev.hmpps.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/csip-api-docs.json
-curl https://alerts-api-dev.hmpps.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/alerts-api-docs.json
-curl https://remand-and-sentencing-api-dev.hmpps.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/court-sentencing-api-docs.json
-curl https://prison-person-api-dev.prison.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/prison-person-api-docs.json
-curl https://personal-relationships-api-dev.hmpps.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/personal-relationships-api-docs.json
-curl https://organisations-api-dev.hmpps.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/organisations-api-docs.json
-curl https://hmpps-person-record-dev.hmpps.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/core-person-api-docs.json
+In the build.gradle.kts there is a `ModelConfiguration` for each of the models.  This contains the URL to be used to
+obtain the JSON Open API specification and also the package name to be used for the generated models.  For each model
+configuration two tasks are created:
+1. `write<ModelName>Json` - this task will download the Open API specification and save it to `openapi-specs`.
+2. `build<ModelName>ApiModel` - this task will generate the models from the specification
+
+So, for example, running
+```shell
+./gradlew clean writeNonAssociationsJson compileKotlin compileTestKotlin
```

Comment:
Different to nomis-update which has `./gradlew clean writeNonAssociationsJson buildNonAssociationsApiModel compileKotlin compileTestKotlin`. Is that on purpose?

Reply:
Also thinking about it I'm not sure you need to clean - that forces all models to be rebuilt. The build model task should have an input which is the json, and therefore will only run if the json has changed.